react/components/OrderInfo/index.tsx

```diff
@@ -38,7 +38,7 @@ const OrderInfo: FunctionComponent<Props> = ({
 
   return (
     <section>
-      <div className="mv6 w-80-ns w-90 center">
+      <div>
```

Comment:
Useful?

Reply:
I had to move this to the `<main>` in order placed, to keep the **banner** aligned as the others elements